src/scss/includes/direction.scss

```diff
@@ -1,6 +1,32 @@
-.direction_panel {
-  background: url("../images/panel_top.svg") #f4f6fa repeat-x !important;
-  background-size: 100% 5px;
+.direction-panel {
+  background-color: $background;
```

Comment:
These 4 rules aren't required at all on desktop now that you reverted to a Panel.
And on mobile, only the `background-color` and shadow are required.

Reply:
some style are needed for mobile through, so i moved them